test/functionalTest/cases/1916_fiware_correlator/fiware_correlator.test

```diff
@@ -36,14 +36,14 @@ brokerStart CB 0
 
```

Comment:
The following file also have an usage of GET /version that should be modified to add `--noPayloadCheck`:

2763_postUpdateContext_error_return/invalid_service_path_gives_400_bad_request.test

```
...
orionCurl --url /version --servicePath "notStartingWithSlash"
...
```

Reply:
I have left this file unmodified intentionally since the expected output is a "BadRequest" response as below:

```json
{
    "description": "Only /absolute/ Service Paths allowed [a service path must begin with /]",
    "error": "BadRequest"
}
```

Reply:
Makes sense. NTC.